src/components/ArticleController.jsx

```diff
@@ -19,17 +19,15 @@ export default class ArticleController extends FalcorController {
       let articleSlug = this.props.params.articleSlug;
       // Access data fetched via Falcor
       const articleData = this.state.data.articlesBySlug[articleSlug];
-      //console.log("Data: " + JSON.stringify(articleData));
       return (
         <div>
-          {/*
-            <div>Controller for article: {articleData.title}</div>
-            <div>Ready?: {this.state.ready ? 'true' : 'false'}</div>
-          */}
           <Article
             title={articleData.title}
+            teaser={articleData.teaser}
+            pubDate={articleData.published_at}
             html={articleData.html}
             authors={articleData.authors}
+            url={"thegazelle.org/issue/" + articleData.issueId + '/' + articleData.category + '/' + articleData.slug}
```

Comment:
For twitter, it just becomes a link within the tweet. Twitter isn't examining the actual link in any way, just displaying it as is, so thegazelle.org/etc/etc will work. For Facebook, the ensure there aren't any issues, I could just add `https://www.` before I concatenate the url when I'm creating the share link. I don't think there would be an issue either way, but just to ensure there isn't.

On that same note, Facebook uses Open Graph tags to help it create the proper share button information anyways. [Ghost adds the OG meta data](https://blog.ghost.org/structured-data/) in automatically, so we shouldn't just get the Gazelle icon for every shared post anymore.